python/lsst/ts/observatory/control/constants/latiss_constants.py

```diff
@@ -0,0 +1,10 @@
+"""Useful constants for running auxTel"""
+from lsst.geom import ExtentD, PointD
+
+plate_scale = 0.1045  # exp.getWcs().getPixelScale().asArcseconds()  # arcsec/pixel
```

Comment:
Please, remove `# exp.getWcs().getPixelScale().asArcseconds() ` from the comment